modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/EditObjectFolderHeader/EditObjectFolderHeader.tsx

```diff
@@ -26,7 +27,14 @@ export default function EditObjectFolderHeader({
 	selectedObjectFolder,
 	setShowModal,
 }: EditObjectFolderHeaderProps) {
-	const [{showChangesSaved}] = useObjectFolderContext();
+	const [{elements, showChangesSaved}, dispatch] = useObjectFolderContext();
+	const [
+		showModalPublishObjectDefinitions,
+		setShowModalPublishObjectDefinitions,
+	] = useState<boolean>(false);
```

Comment:
hi @naraandrade. Could you check if we can add a PublishObjectsDefinitions prop to the ModelBuilderModals interface and use it here instead of creating a new state?